styles/components/_layout.scss

```diff
@@ -0,0 +1,94 @@
+//containers should have one of the standard padding sizes
```

Comment:
That's a good question. Grid should be for only grid classes, and layout should be for everything that isn't part of the grid. 

That got muddy very quickly since some of the flex classes are very useful for non-grid layouts, but it didn't make sense to put them in a different file.

I'll merge them.